Patches/Vanilla Factions Expanded - Pirates/ThingDefs_Misc/ThingDefs_Artillery.xml

```diff
@@ -26,6 +26,13 @@
 
       <!-- Field Gun -->
 
+      <li Class="PatchOperationAdd">
+        <xpath>/Defs/ThingDef[defName = "VFEP_Turret_FieldGun"]</xpath>
+        <value>
+            <fillPercent>0.90</fillPercent>
```

Comment:
the field gun doesn't need to be that tall, it's an artillery piece and won't be blocked by tall buildings